R/zchunk_LA100.IMAGE_downscale_ctry_yr.R

```diff
@@ -30,61 +29,119 @@ module_aglu_LA100.IMAGE_downscale_ctry_yr_DISABLED <- function(command, ...) {
 
     # Load required inputs
     AGLU_ctry <- get_data(all_data, "aglu/AGLU_ctry")
-    IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y <- get_data(all_data, "aglu/IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y")
-    IMAGE_an_FeedIO_Rimg_C_Sys_Y <- get_data(all_data, "aglu/IMAGE_an_FeedIO_Rimg_C_Sys_Y")
-    IMAGE_an_Prodmixfrac_Rimg_C_Y <- get_data(all_data, "aglu/IMAGE_an_Prodmixfrac_Rimg_C_Y")
-
-    # ===================================================
-    # TRANSLATED PROCESSING CODE GOES HERE...
-    #
-    # If you find a mistake/thing to update in the old code and
-    # fixing it will change the output data, causing the tests to fail,
-    # (i) open an issue on GitHub, (ii) consult with colleagues, and
-    # then (iii) code a fix:
-    #
-    # if(OLD_DATA_SYSTEM_BEHAVIOR) {
-    #   ... code that replicates old, incorrect behavior
-    # } else {
-    #   ... new code with a fix
-    # }
-    #
-    #
-    # ===================================================
+    IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y")
+    IMAGE_an_FeedIO_Rimg_C_Sys_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_FeedIO_Rimg_C_Sys_Y")
+    IMAGE_an_Prodmixfrac_Rimg_C_Y <- get_data(all_data, "aglu/IMAGE/IMAGE_an_Prodmixfrac_Rimg_C_Y")
+
+    # Helper function: copy 1970 data to new year 1960
+    extrapolate_1960 <- function(x) {
```

Comment:
Not really extrapolating (or at least way I think of extrapolating) and maybe a more descriptive name, like `extrapolate_1970_to_1960` if you want to stick with extrapolate language or `copy_1970_to_1960` if not. 
Also, if this type of helper function might be more widely useful, it might make sense define externally and make the years an input option, eg
`copy_year1_to_year2 <- function(x, year1, year2){`